tests/basic/upgrade_cluster_test.go

```diff
@@ -89,6 +100,19 @@ var _ = Describe("{UpgradeCluster}", func() {
 					log.Warnf("PDB validation skipped. Current Px-Operator version: [%s], minimum required: [%s]. Error: [%v].", opver, PDBValidationMinOpVersion, err)
 				}
 
+				var vQuorumError error
+				// validate volume quorum during upgrade
+				if opver.GreaterThanOrEqual(ParallelUpgradeOperatorVersion) && pxVersion.GreaterThanOrEqual(ParallelUpgradePXVersion) {
```

Comment:
`if err == nil && opver.GreaterThanOrEqual(PDBValidationMinOpVersion) {` This part needs to go into an "if-else" condition where we check for cluster wide pdb validations if operator version is lesser than 24.2.0 and 24.1.0 or greater. This is already a part of another PR. - https://github.com/portworx/torpedo/pull/2665